benchopt/viz/plot_convergence_curve.py

```diff
@@ -14,7 +14,7 @@ def plot_convergence_curve(df, benchmark):
     Returns
     -------
     fig : instance of matplotlib.figure.Figure
-        The matplotlib figure.
+        The matplotlib figure of the objective values.
```

Comment:
Is it necessary to mention it ? It could be something else

Reply:
For now, the only column plotted is explicitly set as being the 'objective_values'

Reply:
with the stochastic benchmark it will no longer be. for me it does not bring a lot to be more specific here and it will need to be changed soon. "The matplotlib figure with the results" ?